packages/core/admin/admin/src/index.js

```diff
@@ -41,11 +42,12 @@ const run = async () => {
   try {
     const {
       data: {
-        data: { isEE, features },
+        data: { isEE, features, isNpsEnabled },
       },
     } = await get('/admin/project-type');
 
     window.strapi.isEE = isEE;
+    window.strapi.isNpsEnabled = isNpsEnabled
     window.strapi.features = {
       ...window.strapi.features,
       isEnabled: (featureName) => features.some((feature) => feature.name === featureName),
```

Comment:
Maybe I am missing something here but it looks like we handle other features differently than the way you are doing it. Could we keep it consistent?

Reply:
@markkaylor It is the same way as it is done in https://github.com/strapi/strapi/pull/18179 I am fine to change it to a way you think is better but how would that be?

Reply:
I think there's some parallels to be had in https://github.com/strapi/strapi/pull/18179 and this, I think we could organise them both under one umbrella we just need a name...

Reply:
I think that.
flags mind be a good name.

Reply:
```ts
flags: {
	nps: false,
	eePromotions: false
}
```

yeah that could be nice...

Reply:
We probably need Alex to weigh in here as I know he has strong opinions on config structure and naming.

Reply:
@alexandrebodin do you have an opinion here? Let me know if you would like me to change how it's done and I will make the change asap.